### PR TITLE
Build & Rebuild actions are disabled in mx projects!

### PR DESCRIPTION
**Pull request by JaroslavTulach**
A [mistake has been made](https://github.com/apache/netbeans/pull/2822/files#r619574208) when implementing #2822 - now build/rebuild actions are disabled for _mx projects_. This PR fixes that by returning `true` for all supported actions and `false` only otherwise.

When at it I also fixed long opened problem - to run/debug unit test one had to build first, otherwise the changes were not visible - now the build happens before running tests automatically. With 8c3b56d the VSNetBeans UI also knows when run/debug test action is over and can hide the debugging toolbar.

This is not a stopper for 12.4 (who cares about mx projects anyway?), but please consider this PR for inclusion in 12.4, if there is a chance.

### CODE REVIEW

**Comment by JaroslavTulach** [comment]
The tests, including the new [java.mx.project test](https://travis-ci.com/github/apache/netbeans/jobs/500861800#L10021) are all green. Clearly this change must be fixing something.

**Review by dbalek — APPROVED** [review]
Looks fine.

File: java/java.mx.project/src/org/netbeans/modules/java/mx/project/SuiteActionProvider.java
Comment (sdedic) [inline java/java.mx.project/src/org/netbeans/modules/java/mx/project/SuiteActionProvider.java:168]: Should it be `.getDisplayName()` or `.getName()` for the `--only` argument ?

File: java/java.mx.project/src/org/netbeans/modules/java/mx/project/SuiteActionProvider.java
Comment (sdedic) [inline java/java.mx.project/src/org/netbeans/modules/java/mx/project/SuiteActionProvider.java:184]: The calling method (`invokeAction`) checks for `runMx() != null`; this result could trigger `ActionProgress.start` - OK ?

File: java/java.mx.project/src/org/netbeans/modules/java/mx/project/SuiteActionProvider.java
Comment (JaroslavTulach) [reply java/java.mx.project/src/org/netbeans/modules/java/mx/project/SuiteActionProvider.java:168]: `prj.getDisplayName()` doesn't exist. the `name` isn't a project (e.g. suite in mx terms) name, but rather a name of the source group - e.g. project in mx terms.

Comment (sdedic) [reply java/java.mx.project/src/org/netbeans/modules/java/mx/project/SuiteActionProvider.java:168]: Sorry, wrong wording. I wanted to ask: should it be
`..., "build", "--only", name);` or `..., "build", "--only", grp.getName());` ? Since `displayName` can be sometimes `displayName = name + "[" + rel + "]";`

File: java/java.mx.project/src/org/netbeans/modules/java/mx/project/SuiteActionProvider.java
Comment (JaroslavTulach) [reply java/java.mx.project/src/org/netbeans/modules/java/mx/project/SuiteActionProvider.java:184]: It would trigger start, but also `finish(false)`, right? At least that is consistent...

**Comment by JaroslavTulach** [comment]
Waiting for approval from release co-ordinators to merge to `delivery` branch. Or Neil, Geertjan feel free to merge.

**Review by geertjanw — APPROVED** [review]

**Pull request merged**
